### ISSUE TITLE
Add lib64 to setup search path

### ISSUE THREAD

**Issue by mehdisadeghi** [PyTables/PyTables#507]
I have installed hdf5 library in a custom path and I realized that setup script is not able to find the library regardless of setting `HDF5_DIR` or any other environment variable. Looking at the script I added [lib64 folder to search path](https://github.com/PyTables/PyTables/blob/develop/setup.py#L310) and everything worked fine.

Please add `lib64` directory to default search path while looking for `libhdf5` shared object.

**Comment by andreabedini** [comment]
We are supposed to honour HDF5_DIR, any idea what was going wrong?

**Comment by mehdisadeghi** [comment]
@andreabedini Oh, I had missed this notification!

Every time that I try to install pytables on a new machine or on a new virtualenv, I hit this problem. No matter what I do, pytables does not find the library. I have tried exporting HDF5_DIR, exporting CFLAGS, playing with pip options, downloading and installing the package using setup.py, but no luck.

I might have made some mistakes, but the process should be straight forward and not so complicated. The only solution that I found was adding lib64 to pytable search path that fixed the problem for me.
